src/Core/Content/Category/SalesChannel/CategoryRoute.php

```diff
@@ -138,4 +139,42 @@ private function createCriteria(string $pageId, Request $request): Criteria
 
         return $criteria;
     }
+
+    /**
+     * @return array<string, array<string, mixed>>|null
+     */
+    private function buildMergedCmsSlotConfig(CategoryEntity $category, SalesChannelContext $context): ?array
+    {
+        $inheritanceChain = $context->getLanguageIdChain();
+        if (\count($inheritanceChain) <= 1) {
+            return $category->getTranslation('slotConfig');
```

Comment:
line is still not covered by an unit test

Reply:
As I told you before, I didn't check that yet.